### PR TITLE
test: integration test

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSystemTest {
+
+  private static CloudRedisClient client;
+  private static String projectId;
+
+  private static final Logger log =
+      Logger.getLogger(com.google.cloud.redis.it.ITSystemTest.class.getName());
```
Comment (rahulKQL) [inline google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java]: Please use the current package here.

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  public void testGetInstance() {
+    InstanceName name = InstanceName.of(projectId, LOCATION, INSTANCE);
+    Instance response = client.getInstance(name);
+    assertEquals(TIER, response.getTier());
+    assertEquals(MEMORY_SIZE_GB, response.getMemorySizeGb());
+  }
+
+  @Test
+  public void testListInstance() {
+    LocationName parent = LocationName.of(projectId, LOCATION);
```
Comment (dmitry-fa) [inline google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java:86]: testListInstance -> testListInstances

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  private static final Instance.Tier TIER = Instance.Tier.BASIC;
+  private static final String INSTANCE_NAME_PREFIX = "test-instance";
+  private static final String INSTANCE =
+      INSTANCE_NAME_PREFIX + "-" + UUID.randomUUID().toString().substring(0, 8);
+  private static final String LOCATION = "us-central1";
+  private static final int MEMORY_SIZE_GB = 1;
+  private static final String AUTHORIZED_NETWORK = System.getProperty("redis.network", "default");
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
```
Comment (rahulKQL) [inline google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java]: renaming it to `setUp` be more meaningful

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  @Test
+  public void testListInstance() {
+    LocationName parent = LocationName.of(projectId, LOCATION);
+    CloudRedisClient.ListInstancesPagedResponse pagedListResponse = client.listInstances(parent);
+    List<Instance> resources = Lists.newArrayList(pagedListResponse.iterateAll());
+    int instance = 0, count = 0;
+    while (instance < resources.size()) {
+      count++;
+      instance++;
+    }
```
Comment (rahulKQL) [inline google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java:93]: At any point count & instance would hold the same value i.e equal to the resource size. Then why do we need this loop for?

**Review by rahulKQL — CHANGES_REQUESTED** [review]

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java (hunk 4 above)
Comment (pmakani) [reply google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java:93]: if there is more than one instance so we can not set expect value that's why we are counting and it won't be flaky test

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java (hunk 2 above)
Comment (pmakani) [reply google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java:86]: Done.

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java (hunk 1 above)
Comment (pmakani) [reply google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java]: done

File: google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java (hunk 3 above)
Comment (pmakani) [reply google-cloud-redis/src/test/java/com/google/cloud/redis/v1/it/ITSystemTest.java]: done

**Pull request closed without merging**